app/src/main/java/org/gdg/frisbee/android/chapter/InfoFragment.java

```diff
@@ -263,38 +230,53 @@ private Spanned getAboutText(Person person) {
         return Html.fromHtml(aboutText);
     }
 
-    private void updateOrganizersOnline(final Person person) {
```

Comment:
This part is really really hard to review. :( It was a mess before :( But right now it is still complicated.